src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__img-logo {
+    margin: 10px 0 0 20px;
+    padding-right: 30px;
+  }
+
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item {
+    margin-right: 25px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    text-decoration: none;
+    font-family: "Roboto", serif;
+    color: black;
+    font-weight: 500;
+    padding: 25px 0;
+    line-height: 14.06px;
```

Comment:
do not use float numbers to set font height. All numbers in the layout should be an integer. Float numbers appear on the mockup because of conversion errors.